execution_engine/src/core/engine_state/mod.rs

```diff
@@ -1758,14 +1758,13 @@ where
     pub fn get_trie(
         &self,
         correlation_id: CorrelationId,
+        index: u64,
         trie_key: Digest,
-    ) -> Result<Option<Trie<Key, StoredValue>>, Error>
+    ) -> Result<Option<TrieOrChunkedData>, Error>
```

Comment:
Since this is a public method, can you extend its comment - especially explaining what `index` argument stands for.